src/Frame.cc

```diff
@@ -219,25 +219,34 @@ sdf::ElementPtr Frame::Element() const
   return this->dataPtr->sdf;
 }
 
-/////////////////////////////////////////////////
+///////////////////////////////////////////////
```

Comment:
probably accidentally removed 2 characters here

Reply:
True, maybe I'm going too fast over these changes hehe. Thanks for the catch, should be fixed now.